code/__DEFINES/dcs/signals/signals_object.dm

```diff
@@ -503,3 +503,10 @@
 
 /// from /datum/component/dart_insert/on_reskin()
 #define COMSIG_DART_INSERT_PARENT_RESKINNED "dart_insert_parent_reskinned"
+
+/// Fired when the user wants to exit their current object possession. From /proc/release().
+#define COMSIG_END_OBJECT_POSSESSION_VIA_VERB "end_object_possession_via_verb"
+/// Fired when the user exits their current object possession via possessing a new object. From the Initialize() of /datum/component/object_possession
+#define COMSIG_END_OBJECT_POSSESSION_VIA_COMPONENT_CHAIN "end_object_possession_via_chaining"
+/// Fired when the user exits their current object possession via clicking the screen alert. From the Click() of /atom/movable/screen/alert/unpossess_object
+#define COMSIG_END_OBJECT_POSSESSION_VIA_SCREEN_ALERT "end_object_possession_via_screen_alert"
```

Comment:
I agree with melbert these signals suck. If you need to reach out and touch your component it shouldn't BE a component.
Is there a reason to componentize this?

Reply:
yes because we need to hold state in `stashed_name`

Reply:
I can see this being a good component if attached to a mob, a `mob_controlling_object` component, sounds like something that can be re-used down the line. Maybe for a prop hunt ghost game or something.

Reply:
to clarify it seems like this is a pack of data that you want to like, have external control over.
are you sure it's suited to being a component instead of just a datum? would retain the reusability but you wouldn't need signals that are just funcky smelling procs you know?

Reply:
brother i do not want to add another variable to `/mob` the whole point of this is to remove variables

Reply:
ok but why is removing vars something we care about, specially if it makes stuff worse?

Reply:
because it's less not-useful stuff to scroll through on either the in-game view variables panel or the vsc debugger when you're trying to locate something. i forgot to elaborate on this in this PR.

also i'm going with melbert's solution, you won't see any specialized signals once i'm done ironing out the bugs